src/main/scala/viper/gobra/ast/frontend/Ast.scala

```diff
@@ -26,13 +26,15 @@ sealed trait PNode extends Product {
   def pretty(prettyPrinter: PrettyPrinter = PNode.defaultPrettyPrinter): String = prettyPrinter.format(this)
 
   lazy val formatted: String = pretty()
+  lazy val formattedShort: String = pretty(PNode.shortPrettyPrinter)
```

Comment:
Do we need a new prettyPrinter? wouldn't the "normal" pretty printer work given that we are only interested in printing the types correctly?

Reply:
I use the short printer for the symbol table entries